tests/kafkatest/tests/core/zookeeper_security_upgrade_test.py

```diff
@@ -92,7 +93,8 @@ def run_zk_migration(self):
             self.kafka.stop_node(node)
             self.kafka.start_node(node)
 
-    @matrix(security_protocol=["PLAINTEXT","SSL","SASL_SSL","SASL_PLAINTEXT"])
+    @cluster(num_nodes=9)
+    @matrix(security_protocol=["PLAINTEXT", "SSL", "SASL_SSL", "SASL_PLAINTEXT"])
```

Comment:
I think I did here what you suggested in the previous comment, which is use the sasl cluster size for all tests